src/layout/section/slideSection/SlideSection.tsx

```diff
@@ -67,7 +81,8 @@ export const SlideSection = (props: SlideSectionProps): JSX.Element => {
   const [expandedHeight, setExpandedHeight] = useState<number>(0)
   useEffect(() => {
     // Reduced height is the height of reducedContent element + GripHandle height
-    const reducedContentHeight = reducedContentRef.current.clientHeight
+    // On small screen, we only show a 48px height element for touch control
+    const reducedContentHeight = isScreenTooSmall ? 16 : reducedContentRef.current.clientHeight
```

Comment:
use constant

Reply:
no magic numbers.. why `16`?

Reply:
Reason was to have a bit more height to the reduced element so you can manipulate it with touch events with ease. I'll change that so it's not a magic number.